metrics/core/metrics.go

```diff
@@ -1008,17 +1008,17 @@ var MetricDiskIORead = Metric{
 		Type:        MetricCumulative,
 		ValueType:   ValueInt64,
 		Units:       UnitsBytes,
-		Labels:      metricLabels,
+		Labels:      diskLabels,
 	},
 	HasLabeledMetric: func(spec *cadvisor.ContainerSpec, stat *cadvisor.ContainerStats) bool {
 		return spec.HasDiskIo
 	},
 	GetLabeledMetric: func(spec *cadvisor.ContainerSpec, stat *cadvisor.ContainerStats) []LabeledMetric {
 		result := make([]LabeledMetric, 0, len(stat.DiskIo.IoServiceBytes))
 		for _, ioServiceBytesPerPartition := range stat.DiskIo.IoServiceBytes {
-			resourceIDKey := ioServiceBytesPerPartition.Device
-			if resourceIDKey == "" {
-				resourceIDKey = fmt.Sprintf("%d:%d", ioServiceBytesPerPartition.Major, ioServiceBytesPerPartition.Minor)
+			diskName := ioServiceBytesPerPartition.Device
+			if diskName != DevSda {
```

Comment:
can we confirm that this is the same on ubuntu?

Reply:
Verified.